tools/ClassGen/NodeBuilder.cpp

```diff
@@ -624,6 +723,22 @@ void NodeBuilder::emitImportMethods(std::ostream &os) const {
          << ", dict));\n\n";
     }
   }
+  for (const auto &op : variableOutputMembers_) {
+    auto ty = getCtorArgTypename(MemberType::Unsigned);
+    os << "  " << ty << " " << op.second << "_Size;\n"
```

Comment:
We usually don't use underscores in the middle of a name.

Reply:
Fixed.